x-pack/plugins/security_solution/common/endpoint/data_generators/exceptions_list_item_generator.ts

```diff
@@ -256,28 +256,36 @@ export class ExceptionsListItemGenerator extends BaseDataGenerator<ExceptionList
     const os = this.randomOSFamily() as ExceptionListItemSchema['os_types'][number];
     const entriesList: CreateExceptionListItemSchema['entries'] = [
       {
-        field: 'file.path.text',
+        field: 'file.executable.caseless',
+        value:
+          os === 'windows'
+            ? ['C:\\some\\path', 'C:\\some\\other\\path', 'C:\\yet\\another\\path']
+            : ['/some/path', 'some/other/path', 'yet/another/path'],
+        type: 'match_any',
         operator: 'included',
-        type: 'wildcard',
-        value: os === 'windows' ? 'C:\\Fol*\\file.*' : '/usr/*/*.dmg',
       },
       {
-        field: 'file.path.text',
+        field: 'file.hash.sha256',
+        value: [
+          'a665a45920422f9d417e4867efdc4fb8a04a1f3fff1fa07e998e86f7f7a27ae3',
+          '2C26B46B68FFC68FF99B453C1D30413413422D706483BFA0F98A5E886266E7AE',
+          'FCDE2B2EDBA56BF408601FB721FE9B5C338D10EE429EA04FAE5511B68FBF8FB9',
+        ],
+        type: 'match_any',
         operator: 'included',
-        type: 'wildcard',
-        value: os === 'windows' ? 'C:\\Fol*\\file.exe' : '/usr/*/app.dmg',
       },
       {
-        field: 'process.executable.caseless',
-        value:
-          os === 'windows'
-            ? ['C:\\some\\path', 'C:\\some\\other\\path', 'C:\\yet\\another\\path']
-            : ['/some/path', 'some/other/path', 'yet/another/path'],
+        field: 'file.hash.md5',
+        value: [
+          'a665a45920422f9d417e4867efdc4fb8a04a1f3fff1fa07e998e86f7f7a27ae3',
```

Comment:
just suggestion: might want to use actual "real" hashes in these, so that if we attempt to use this generator with FTR tests, it will generate validate API values